Chessington.GameEngine/Helpers.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Chessington.GameEngine
+{
+    public class Helpers
+    {
+        public void AddMoves(
+            Board board, 
+            List<Direction> directions, 
+            Square currentSquare, 
+            int spacesAway, 
+            List<Square> moves, 
+            Player player
+            )
+        {
+            var newDirections = new List<Direction>();
+            foreach (var direction in directions)
+            {
+                var potentialMove = Square.At(
+                    currentSquare.Row + spacesAway * direction.RowOffset,
+                    currentSquare.Col + spacesAway * direction.ColOffset
+                    );
```

Comment:
Same minor thing about the indentation here. I know Visual Studio puts it here by default but I think it's more readable when it's indented the same as the start of the statement, i.e.
``` 
var potentialMove = Square.At(
  currentSquare.Row...,
  ...
);
```
🙂